src/engine/enginebuffer.h

```diff
@@ -77,11 +79,17 @@ class EngineBuffer : public EngineObject {
 
     // This enum is also used in mixxx.cfg
     // Don't remove or swap values to keep backward compatibility
-    enum KeylockEngine {
-        SOUNDTOUCH,
-        RUBBERBAND_FASTER,
-        RUBBERBAND_FINER,
-        KEYLOCK_ENGINE_COUNT,
+    enum class KeylockEngine : std::uint8_t {
+        SoundTouch,
+        RubberBandFaster,
+        RubberBandFiner,
+    };
```

Comment:
Come to think of it, the explicit type annotation is probably unnecessary. We should instead explicitly specify the underlying values to emphasize that they need to be stable. 
```suggestion
    enum class KeylockEngine {
        SoundTouch = 0,
        RubberBandFaster = 1,
        RubberBandFiner = 2,
    };
```

Reply:
fixed https://github.com/ronso0/mixxx/pull/27/commits/56c6c9d3e642da47cd9f4ec3d497fb5d74584888